src/strategies/decentraland-estate-size/index.ts

```diff
@@ -0,0 +1,34 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = '2fd';
+export const version = '0.1.0';
+
+const abi = [
+  'function getLANDsSize(address _owner) public view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const multipler = options.multiplier || 1
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const response = await multicall(
+    network,
+    provider,
+    abi,
+    addresses.map((address: any) => [options.address, 'getLANDsSize', [address]]),
```

Comment:
Do you have this data on a subgraph? Somehow this call is taking lot of resources on our node, does this call have complex logic on the smart contract? With this implementation the multicall request may fail if there is too much addresses (~350).

Reply:
Yes, it has a complex logic. I'm going to reimplement it using our graph 😄